_component_design/help-beacon.md

```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
+![]({{site.url}}/assets/images/components/help/beacon/beacon-hero.svg){:width="80%"}
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Best practice
+
+- Place the beacon next to (left or right) navigation items to show location and label.
+- Place the beacon next to labels or section headers.
+- Use the beacon in conjuction with the directional pop-over.
+- **Do not** use the beacon in leiu of UX best practices with regard to affordance.
+- **Do not** use other forms of notification to deliver the context such as automatic tool tips or directional pop-overs
+- **Do not** obscure the item of text label with the beacon.
+
+{% endcolumn %}
 
-<div class="ws-beacons">
-  <div class="ws-beacon-con">
-    <p>Grey:</p>
-    <div class="ds-beacon-con"><span class="ds-beacon ds-tooltip-source temp-beacon" title="<h3 class='ds-tooltip-h3'>New Feature Title</h3><p class='ds-tooltip-p'>A very short explanation of the new feature and something about what it does.</p><div class='ds-btn-group-tooltip'><button class='ds-btn-sm ds-tooltip-btn' onclick='removeTooltips()' type='button'>Learn More</button><button class='ds-btn-sm-sec ds-tooltip-btn' onclick='removeTooltips()' type='button'>Cancel</button></div>"></span></div>
-  </div>
-  <div class="ws-beacon-con">
-    <p>Theme match:</p>
-    <div class="ds-beacon-con"><span class="ds-beacon-cyan ds-tooltip-source temp-beacon" title="<h3 class='ds-tooltip-h3'>New Feature Title</h3><p class='ds-tooltip-p'>A very short explanation of the new feature and something about what it does.</p><div class='ds-btn-group-tooltip'><button class='ds-btn-sm ds-tooltip-btn' onclick='removeTooltips()' type='button'>Learn More</button><button class='ds-btn-sm-sec ds-tooltip-btn' onclick='removeTooltips()' type='button'>Cancel</button></div>"></div>
-  </div>
-  <div class="ws-beacon-con">
-    <p>Theme complement:</p>
-    <div class="ds-beacon-con"><span class="ds-beacon-orange ds-tooltip-source temp-beacon" title="<h3 class='ds-tooltip-h3'>New Feature Title</h3><p class='ds-tooltip-p'>A very short explanation of the new feature and something about what it does.</p><div class='ds-btn-group-tooltip'><button class='ds-btn-sm ds-tooltip-btn' onclick='removeTooltips()' type='button'>Learn More</button><button class='ds-btn-sm-sec ds-tooltip-btn' onclick='removeTooltips()' type='button'>Cancel</button></div>"></div>
-  </div>
 </div>
+
+### Default layout
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+This example shows the multiple states for the beacon. The beacon will animate on a loop to catch the user’s eye. Once the beacon is clicked, it dissapears. The pop-over could generate another beacon to provide a tour experience for more advanced workflows or features.
```

Comment:
The last sentence here about the tour experience seems out of place. Move it to the next paragraph and condense with the first sentence there.

Reply:
Now reads

```
This example shows the multiple states for the beacon. The beacon animates on a loop to catch the user’s eye. Once the user clicks the beacon, it disappears. 

To create a tour experience for more advanced features, string together beacons. In this scenario, beacons appear one at a time. Ensure you prioritize demonstrating the most important or impactful feature first. Provide an opt-out option on each [popover]({{site.baseurl}}/components/popovers.html) at every step in the tour. Ideally, track the clicks and drop-offs throughout the tour to improve usefulness for the end user.
```